client/core/core.go

```diff
@@ -3581,14 +3637,15 @@ func (c *Core) resumeTrades(dc *dexConnection, trackers []*trackedTrade) assetMa
 		// If matches haven't redeemed, but the counter-swap has been received,
 		// reload the audit info.
 		isActive := tracker.metaData.Status == order.OrderStatusBooked || tracker.metaData.Status == order.OrderStatusEpoch
-		var needsCoins bool
+		var matchesNeedingCoins []*matchTracker
 		for _, match := range tracker.matches {
 			dbMatch, metaData := match.Match, match.MetaData
-			var needsAuditInfo bool
+			var needsCoins, needsAuditInfo bool
```

Comment:
`needsCoins` is only used for logging now, it appears. Could just use `len(matchesNeedingCoins) > 0`.